config/application.php

```diff
@@ -95,6 +95,7 @@
 /**
  * Custom Settings
  */
+Config::define('DISALLOW_INDEXING', WP_ENV !== 'production');
```

Comment:
```suggestion
Config::define('DISALLOW_INDEXING', true);
```

Then add `Config::define('DISALLOW_INDEXING', false);` to `config/environments/staging.php` and `config/environments/development.php`

---

Alternatively:

```suggestion
Config::define('DISALLOW_INDEXING', env('DISALLOW_INDEXING') ?? WP_ENV !== 'production');
```